mteb/tasks/Audio/Any2AnyRetrieval/CommonVoice.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import annotations
+
+from mteb.abstasks.Image.AbsTaskAny2AnyRetrieval import AbsTaskAny2AnyRetrieval
+from mteb.abstasks.TaskMetadata import TaskMetadata
+import datasets
+from datasets import Dataset, DatasetDict, load_dataset
+from tqdm import tqdm
+
+class CommonVoiceA2TRetrieval(AbsTaskAny2AnyRetrieval):
```

Comment:
Let's specify version to be clear. Also, this should inherit from MultilingualTask.
```suggestion
from mteb.abstasks.MultilingualTask import MultilingualTask

class CommonVoice17A2TRetrieval(AbsTaskAny2AnyRetrieval, MultilingualTask):
```

Reply:
Renamed.